scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,14 +71,41 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
-do
-    case "${flag}" in
-          b) bundle_id="$OPTARG";;
-          t) dest_dir="$OPTARG";;
```

Comment:
These two options seems removed from here but seems not added below in while loop, are we removing those permanently ?  
seems those are still showing up in the usage but not captured if someone provide it on command line.

Reply:
those are now passional argument
Initially it was like
"/opt/seagate/cortx/motr/libexec/m0reportbug-bundler -b SB999901_7020210e0f264211b487e46e691cc536_motr -t /var/log/cortx/support_bundle/"

but now it is
/opt/seagate/cortx/motr/libexec/m0reportbug-bundler SBbw5f7n9i /var/log/support_bundle